tests/posix/common/src/pthread.c

```diff
@@ -371,6 +377,67 @@ ZTEST(pthread, test_pthread_termination)
 	zassert_equal(ret, ESRCH, "cancelled a terminated thread!");
 }
 
+ZTEST(pthread, test_pthread_tryjoin)
+{
+	pthread_t th = {0};
+	void *retval;
+
+	/* Creating a threads that exits after 1s*/
+	zassert_ok(pthread_create(&th, NULL, timedjoin_thread, NULL));
+	usleep(USEC_PER_MSEC * 100U);
+
+	/* Attempting to join immediately should fail */
```

Comment:
```suggestion
	/* Attempting to join without blocking should fail */
```
I was wondering why the comment says 'join immediately' when we slept for 100ms above

Reply:
Fair enough. I adjusted to `/* Attempting to join, when thread is still running, should fail */`, what do you think?